core/src/main/java/org/stellar/anchor/auth/JwtService.java

```diff
@@ -102,6 +102,21 @@ public String encode(Sep24MoreInfoUrlJwt token) throws InvalidConfigException {
     return builder.signWith(SignatureAlgorithm.HS256, sep24MoreInfoUrlJwtSecret).compact();
   }
 
+  public String encode(ApiAuthJwt token) throws InvalidConfigException {
+    if (platformAuthSecret == null) {
+      throw new InvalidConfigException(
+          "Please provide the secret before encoding JWT for API Authentication");
+    }
+
+    Calendar calNow = Calendar.getInstance();
+    Calendar calExp = Calendar.getInstance();
+    calExp.setTimeInMillis(calNow.getTimeInMillis() + 1000L * token.getExp());
+    JwtBuilder builder =
+        Jwts.builder().setIssuedAt(calNow.getTime()).setExpiration(calExp.getTime());
+
+    return builder.signWith(SignatureAlgorithm.HS256, platformAuthSecret).compact();
```

Comment:
Doesn't the secret we use depend on whether `ApiAuthJwt` is for callback or platform?